LogLine.cpp

```diff
@@ -0,0 +1,84 @@
+#include "LogLine.h"
+#include <iostream>
+
+using namespace std;
+
+LogLine::LogLine(string line) {
+    doParsing(line);
+}
+
+void LogLine::doParsing(string input) {
+    int set_count = 0, startIndex = 0;
+    string word;
+    for (int i = 0; i < input.length(); i++) {
+        if (input[i] == ' ') {
+            word = input.substr(startIndex, i - startIndex);
```

Comment:
add a logic to consider whole remaining line for LOG Message

Reply:
done